test/AWS.Deploy.CLI.UnitTests/Utilities/InMemoryInteractiveService.cs

```diff
@@ -0,0 +1,160 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace AWS.Deploy.CLI.UnitTests.Utilities
```

Comment:
can we move it to some shared project may be `AWS.Deploy.Tests.Common`?

Reply:
Done